libr/anal/fcn.c

```diff
@@ -468,6 +468,16 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 		return R_ANAL_RET_ERROR; // MUST BE TOO DEEP
 	}
 
+	RListIter *iter;
+	RAnalPlugin *plugin;
+	r_list_foreach (anal->plugins, iter, plugin) {
+		if (plugin->is_valid_offset) {
+			int r = plugin->is_valid_offset(anal, addr, 0);
+			if (r < 0) return r;
```

Comment:
missing braces, you can just if (plugin->is_valid_offset) as long as the return value is not used by anything else